examples/tutorials/sklean_models.ipynb

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
<p>All the tutorials have a number in its name which helps to group them thus allowing to learn them in a more structured way.</p><p>So think thoroughly in what group to put this tutorial. The possible candidates are 03, 05, or you can create a separate group if it makes any sense at all.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>